rfc-079-script-imports-of-production-data.md

```diff
@@ -0,0 +1,56 @@
+# Script imports of production data
+
+## Summary
+
+Developers writing production data to file on a local or remote server MUST do
+this via an automated script, rather than doing this manually, e.g. via the
+Rails console.
+
+This script MUST anonymise any sensitive data, and the data dump MUST be stored
+outside of the working repository to ensure it can't be inadvertedly committed
+alongside any changes to the codebase.
+
+## Problem
+
+Fetching data from a database by hand is error-prone. A data dump may be left
+on a server, committed to a repository or even in some cases alter production
+data in unforeseen ways. Developers may not realise they have exposed or
+impacted production data until long after an incident occurs.
+
+By writing a script to automate this process, we ensure that every import of
+data from a production environment is done in the same way, minimising room for
+error. It also allows us to scrub the resulting data dump of any
+personally-identifying and security-sensitive information.
+
+Committing a data-fetching script to the repository makes it visible to other
+members of the team in the code review process, and allows others to provide
+feedback to ensure we're reaching a consensus on the safest way of fetching
+this data.
+
+## Proposal
+
+If developing an application requres a developer to write any production data
+containing sensitive data to a local file or a file on a remove server, this
+MUST be done via an automated script.
+
+The script MUST anonymise any personally-identifying or security-sensitive
```

Comment:
I'm not sure about this being a MUST. I worry it'll be a PITA for some edge cases, like those where email address ends up being an important identifier within the system.

Technically, this guidance applies to something like extracting data from eLinks to send to eJudiciary. If we anonymised that data, it'd be useless.

How about:

> The script MUST remove any security-sensitive data. The script SHOULD remove or anonymise any personal data.